### PR TITLE
wl-mirror: Install provided helper script

### PR DESCRIPTION
**Pull request by vutunganh**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by vutunganh** [comment]
Only makes sense with https://github.com/void-linux/void-packages/pull/42495 being merged first.

File: srcpkgs/wl-mirror/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Optionally install 'pipectl', 'slurp', and either 'rofi' or 'menu' to
+use the helper wl-present script.
```
Comment (classabbyamp) [inline srcpkgs/wl-mirror/INSTALL.msg:2]: does this script have upstream documentation of these dependencies? if so, i don't see a need for this. if not, this should be a README.voidlinux not INSTALL.msg

**Comment by classabbyamp** [comment]
> Only makes sense with https://github.com/void-linux/void-packages/pull/42495 being merged first.

then it makes sense to bring that change into this PR

File: srcpkgs/wl-mirror/INSTALL.msg (hunk 1 above)
Comment (vutunganh) [reply srcpkgs/wl-mirror/INSTALL.msg:2]: I would say so: https://github.com/Ferdi265/wl-mirror#script-dependencies.

**Comment by vutunganh** [comment]
> > Only makes sense with #42495 being merged first.
> 
> then it makes sense to bring that change into this PR

Ok, I have just done that.

File: srcpkgs/wl-mirror/INSTALL.msg (hunk 1 above)
Comment (vutunganh) [reply srcpkgs/wl-mirror/INSTALL.msg]: I have just removed the file.

File: srcpkgs/wl-mirror/template [hunk 2]
```diff
@@ -3,7 +3,9 @@ pkgname=wl-mirror
 version=0.12.2
 revision=1
 build_style=cmake
-configure_args="-DWLR_PROTOCOL_DIR=${wrksrc}/proto/wlr-protocols -DINSTALL_DOCUMENTATION=ON"
+configure_args="-DWLR_PROTOCOL_DIR=${wrksrc}/proto/wlr-protocols \
+ -DINSTALL_EXAMPLE_SCRIPTS=ON \
+ -DINSTALL_DOCUMENTATION=ON"
```
Comment (sgn) [inline srcpkgs/wl-mirror/template:8]:
```suggestion
configure_args="-DWLR_PROTOCOL_DIR=${wrksrc}/proto/wlr-protocols
 -DINSTALL_EXAMPLE_SCRIPTS=ON
 -DINSTALL_DOCUMENTATION=ON"
```

Comment (vutunganh) [reply srcpkgs/wl-mirror/template]: Oops, my bad. Should be fixed.

**Pull request merged**